monai/bundle/scripts.py

```diff
@@ -1229,8 +1185,10 @@ def onnx_export(
         copy_model_state(dst=net, src=ckpt if key_in_ckpt_ == "" else ckpt[key_in_ckpt_])
 
     converter_kwargs_.update({"inputs": inputs_, "use_trace": use_trace_})
-    onnx_model = convert_to_onnx(model=net, **converter_kwargs_)
-    onnx.save(onnx_model, filepath_)
+
+    onnx_saver = OnnxSaver()
```

Comment:
This is what I would call a poltergeist, an object with no state and an ephemeral life span which disappears after a task is done.